code/model/DMSDocumentSet.php

```diff
@@ -1,19 +1,41 @@
 <?php
 /**
  * A document set is attached to Pages, and contains many DMSDocuments
+ *
+ * @property Varchar Title
+ * @property  Text KeyValuePairs
+ * @property  Enum SortBy
+ * @property Enum SortByDirection
  */
 class DMSDocumentSet extends DataObject
 {
+
+    /**
+     * This gets switched up during testing to mock the DMSDocument
+     *
+     * @var varchar
+     */
+    public static $linked_document_type = 'DMSDocument';
+
     private static $db = array(
-        'Title' => 'Varchar(255)'
+        'Title' => 'Varchar(255)',
+        'KeyValuePairs' => 'Text',
+        "SortBy" => "Enum('LastEdited,LastChanged,Created,Title')')",
+        "SortByDirection" => "Enum('DESC,ASC')')",
```

Comment:
@robbie The desired sort order per set needs to be persisted so it's a need yes.